docs/supervisor-deploy-with-image-from-private-registry.md

```diff
@@ -19,13 +19,15 @@
 Overview
 ========
 
-In Air-gapped environment, users usually would like to deploy application from private registry which is protected by registry credential. To meet this requirement, we support to deploy Velero in Supervisor Cluster with images from private registry.
+In Air-gapped environment, users usually would like to deploy application from private registry which is protected by registry credential. To meet this requirement, we support deploying Velero in Supervisor Cluster with images from private registry.
 
 Manifest
 ========
 
 *   Velero vSphere operator YAML: [https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml)
 
+Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version,  please use this [YAML file](https://vmwaresaas.jfrog.io/ui/native/Velero-YAML/Velero/SupervisorService/1.4.0/velero-vsphere-1.4.0-def.yaml).
```

Comment:
Actually I think it's better to ask them to find the latest version from this page as 1.4.0 won't be the latest after we release 1.5: https://github.com/vsphere-tmm/Supervisor-Services/blob/main/README.md#velero-versions

Reply:
Fixed.